src/HonzaBotner.Discord/Command/Permissions/Entities/PermissionEntry.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace HonzaBotner.Discord.Command.Permissions.Entities
+{
+    public class PermissionEntry
+    {
+        public Guid Id { get; set; }
+
+        [Required] public PermissionEntryType Type { get; set; }
```

Comment:
nit: I am used to having the attribute on its own line. Why did you write it like that? Not strongly against it, if you have some reason for it.

Reply:
Rider autoformat goes brrr